adyenv6core/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -698,6 +693,17 @@ public void initializeCheckoutData(Model model) {
         modelService.save(cartModel);
     }
 
+    private boolean filterPaymentMethods(PaymentMethod paymentMethod) {
+
+        return (! "scheme".equals(paymentMethod.getType())
+                && ! "bcmc".equals(paymentMethod.getType())
+                && ! "bcmc_mobile_QR".equals(paymentMethod.getType())
+                && (! "wechatpay".contains(paymentMethod.getType()) && "wechatpayWeb".equals(paymentMethod.getType()))
+                && ! "wechatpayQR".equals(paymentMethod.getType())
```

Comment:
I think this line can be removed since wechatpayQR is filtered out in the previous line

Reply:
Correct